src/backend/utils/storage.js

```diff
@@ -1,39 +1,73 @@
+const crypto = require('crypto');
+const normalizeUrl = require('normalize-url');
+const { logger } = require('./logger');
 const { redis } = require('../lib/redis');
 
+const feedNamespace = 't:feed:';
+const postNamespace = 't:post:';
+
+const createKey = (url, prefix) => {
+  try {
+    return prefix.concat(
+      crypto
+        .createHash('sha256')
+        .update(url)
+        .digest('base64')
+    );
+  } catch (error) {
+    logger.error(`There was an error processing ${url}`);
+    throw error;
+  }
+};
+
+const createPostKey = uri => {
+  return createKey(normalizeUrl(uri), postNamespace);
+};
+
+const createFeedKey = uri => {
+  return createKey(uri, feedNamespace);
+};
+
 // Redis Keys
-const FEED_ID = 'feed_id';
-const FEEDS = 'feeds';
+const FEEDS = 't:feeds';
```

Comment:
Let's move these two key names up to the top where you define your other namespaces, and make the names more consistent.  You have `feedNamespace` and `postNamespace`, which is good.  Let's rename `POSTS` to `postsKey` and `FEEDS` to `feedsKey`, so it's clear what all of these things are.